### PR TITLE
Qweb3 dist changes

### PR DESCRIPTION
**Pull request by dwalintukan**
Note I added babel dependencies as qweb3 is using latest babel plugins. And added `npm start` command so it can run the babel-node command. Use this instead of `node src/index.js`

### CODE REVIEW

**Comment by dwalintukan** [comment]
Yup holding off until the demo is done. Changes are using new babel syntax now.

File: src/sync/index.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
-const _ = require('lodash');
+import _ from 'lodash';
+import { Qweb3, Contract } from 'qweb3';
```
Comment (chrisli30) [inline src/sync/index.js]: Yeah this is what I meant in last review, however, the way Qweb3 is referenced is not conventional. 
In Qweb3 src/index.js, Qweb3 as the main module, should be exported using "default export", and Contract, as a submodule, is "export", thus here we referencing them by "import Qweb3, {Contract} from 'qweb3'".

The same use pattern as import React, { Component } from 'react';

File: src/sync/models/centralizedOracle.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
-const _ = require('lodash');
-const qDecoder = require('qweb3/src/decoder');
-const utils = require('qweb3/src/utils');
+import _ from 'lodash';
+import { Decoder, Utils } from 'qweb3';
```
Comment (chrisli30) [inline src/sync/models/centralizedOracle.js]: Those two follows right patterns.

File: src/sync/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/sync/index.js]: ah I see what you mean. Qweb3 should be the default export. then Contract, Decoder, etc will be component imports.

Comment (dwalintukan) [reply src/sync/index.js]: But the way that qweb3 is setup now, `Qweb3` class is entirely independent from `Contract` class. Shouldn't they all be considered individual components?

Comment (chrisli30) [reply src/sync/index.js]: I think you mean they are unrelated. Yeah, to people who use this library they wouldn't know whether components would relate to each other or not. My point is that libraries usually follow a pattern where there is a default export and multiple module exports.

Comment (dwalintukan) [reply src/sync/index.js]: 👍 fixed

**Pull request merged**